modules/serverless-logic-upgrade-1.36-finalizing-job-service.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-finalizing-job-service_{context}"]
+= Finalizing the Job Service upgrade
+
+After the {ServerlessOperatorName} is upgraded to version 1.36.0, a new ReplicaSet for the Job Service component is created automatically. To clean up older resources, you can delete the Job Service ReplicaSets created by the previous 1.35.0 version.
+
+.Procedure
+
+. List all ReplicaSets for the Job Service by running the following command:
```

Comment:
```suggestion
. (Optional) Identify and the delete the old `ReplicaSet` running the following commands:
```

Reply:
@wmedvede 

For listing, the command is different. 
`oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>`

For deleting the command is different: 
`oc delete replicaset <old_replicaset_name> -n <target_namespace>`

Then why are we removing these steps?

Reply:
@kaldesai the case of the jobs service is analogous to the DataIndex ReplicasSet removal.
See attached screenshot.

People can optionally delete.